docs/reference/faq.md

```diff
@@ -103,7 +103,7 @@ Other features, like the new _wide_ and _full-wide_ alignment options, will simp
 
 ## How will editor styles work?
 
-Themes can provide editor styles for blocks by using the following hook:
+Refular editor styles will work as is in most casees and themes can also load extra stylesheets by using the following hook:
```

Comment:
Typo: "regular"

Reply:
Another typo: "casees" instead of "cases".